src/utils/input_validation/validators.py

```diff
@@ -0,0 +1,177 @@
+import os, sys, uuid 
```

Comment:
Two of the imported libraries (os and sys) are not actually used in the code, so I'd suggest removing them.